sdks/wasm/Makefile

```diff
@@ -410,13 +416,17 @@ test-debugger: build-debugger-test-app build-dbg-testsuite
 run-browser-tests: $(BROWSER_TEST)/.stamp-browser-test-suite
 	(cd $(BROWSER_TEST) && npm test)
 
+run-browser-threads-tests: $(BROWSER_TEST_THREADS)/.stamp-browser-test-threads-suite
+	(cd $(BROWSER_TEST_THREADS) && npm test)
+
 clean: clean-sdk
```

Comment:
Maybe run clean-browser-tests here as well and get rid of the duplication in the commands